tests/selection.spec.ts

```diff
@@ -347,3 +350,20 @@ test('select text leaving a few words in the last line and delete', async ({
   const textOne = await getQuillSelectionText(page);
   expect(textOne).toBe('abc89\n');
 });
+
+test('Click the list icon to select', async ({ page }) => {
+  await enterPlaygroundRoom(page);
+  await initEmptyState(page);
+  await initThreeList(page);
+  await assertRichTexts(page, ['123', '456', '789']);
+  await page.mouse.click(100, 165);
```

Comment:
Using coordinates directly is unreliable, can you change to use [locators](https://playwright.dev/docs/locators)?

Reply:
ok i will try to use it